library/ui/src/main/java/com/google/android/exoplayer2/ui/PlayerControlView.java

```diff
@@ -196,6 +197,9 @@
   public static final int MAX_WINDOWS_FOR_MULTI_WINDOW_TIME_BAR = 100;
 
   private static final long MAX_POSITION_FOR_SEEK_TO_PREVIOUS = 3000;
+  private static final int DEFAULT_UPDATE_DP = 5;
+  private static final long MIN_UPDATE_FREQUENCY_MS = 50;
```

Comment:
It seems we already use a minimum delay of 200ms for the cases where the playback speed is > 5.0f. Can you change that accordingly, and also use the same constants in updateProgress()?

Reply:
Not sure if I understood correctly. Should I change MIN_UPDATE_FREQUENCY_MS to 200ms? The other half is done.

Reply:
Thinking about it again, 50ms might actually be a better minimum value. :) 
After your change. we'll only use such low values for very short video durations and given that the human eye can easily see 20 updates per second, this seems like a good value to get smooth movements.